### PR TITLE
Book store setup

### PR DESCRIPTION
**Pull request by Tobby8629**
create the bookstore set up:
- create the bookstore for mobile and desktop screen
- use min-width for responsiveness
- create the bookstore pages (books and categories)
- create the navbar
- A form to add books
- React hook to manage state (useState)

### CODE REVIEW

File: src/component/Categories.jsx [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+function Categories() {
+  return (
+    <div>
+      Categories
+    </div>
```
Comment (AnselemOdims) [inline src/component/Categories.jsx]: - [ ] Good job working on the categories page, but please kindly ensure it has a button with the text `Check Status`

File: src/component/Book.jsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+            <h3>{book.title}</h3>
+            <p>{book.author}</p>
+          </div>
+
+          <div className="btns">
+            <button type="submit">comments</button>
+            <button type="submit">remove</button>
+            <button type="submit">Edit</button>
+          </div>
+        </div>
```
Comment (AnselemOdims) [inline src/component/Book.jsx]: - [ ] Good job working on this, but in order to make the component more modularized and reusable please kindly make this a separate `Book` component which can then be imported and used here.

**Review by AnselemOdims — CHANGES_REQUESTED** [review]
### Hi @Tobby8629 , 
You have been doing a tremendous job so far :ok_hand:
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
PR has a title and summary. :heavy_check_mark:
No linter errors :heavy_check_mark:

## Required Changes ♻️
- [ ] Please also check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity! You can tag me using @AnselemOdims for a quicker response

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by Deepakdanger — APPROVED** [review]
Hi @Tobby8629, 

### Status: Approved ✔️

Your project is complete!  There is nothing else to say other than... it's time to merge it  🚢 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Tobby8629** [comment]
Thank you for
The review

**Pull request merged**